examples/backend-adapters/server/src/effect.ts

```diff
@@ -1,46 +1,59 @@
-import { HttpServer } from "@effect/platform";
-import { BunHttpServer, BunRuntime } from "@effect/platform-bun";
-import { Effect, Layer } from "effect";
+import "dotenv/config";
```

Comment:
I don't think this is actually needed tbh

Reply:
Tax doesnt load process.env automatically I dont think